components/automate-ui/src/app/components/resource-dropdown/resource-dropdown.component.spec.ts

```diff
@@ -1,21 +1,22 @@
-import { CUSTOM_ELEMENTS_SCHEMA, EventEmitter } from '@angular/core';
+import { CUSTOM_ELEMENTS_SCHEMA, EventEmitter, SimpleChange } from '@angular/core';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { FormsModule } from '@angular/forms';
 
+import { using } from 'app/testing/spec-helpers';
 import { ChefPipesModule } from 'app/pipes/chef-pipes.module';
-import { ResourceDropdownComponent } from './resource-dropdown.component';
+import { ResourceDropdownComponent, ResourceChecked } from './resource-dropdown.component';
 
 describe('ResourceDropdownComponent', () => {
   let component: ResourceDropdownComponent;
   let fixture: ComponentFixture<ResourceDropdownComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ ResourceDropdownComponent ],
-      imports: [ ChefPipesModule, FormsModule ],
-      schemas: [ CUSTOM_ELEMENTS_SCHEMA ]
+      declarations: [ResourceDropdownComponent],
+      imports: [ChefPipesModule, FormsModule],
+      schemas: [CUSTOM_ELEMENTS_SCHEMA]
```

Comment:
At some point, we could introduce a bunch of `MockComponents` to get rid of this, I guess? Or has our stance towards this changed?

Reply:
Yep, should do that.